src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -63,10 +68,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String userEmail = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), "", Whitelist.none(), new OutputSettings().prettyPrint(false));
 
+    MutableDataSet options = new MutableDataSet();
+    // Set the root path for emoji image files
+    options.set(EmojiExtension.ROOT_IMAGE_PATH, "https://www.webfx.com/tools/emoji-cheat-sheet/graphics/emojis/");
```

Comment:
"https://www.webfx.com/tools/emoji-cheat-sheet/graphics/emojis/" can be extracted out as a constant